actions/changelog-generate/action.yaml

```
@@ -75,12 +75,23 @@ runs:
         echo  "fromtag=$setfrom">> $GITHUB_OUTPUT
       shell: bash
 
+    - name: Download default changelog
+      id: config
+      run: |
+        if [[ -n "${{ inputs.config }}" ]]; then
+          echo  "path=${{ inputs.config }}">> $GITHUB_OUTPUT
+        else
+          wget -c https://raw.githubusercontent.com/bakdata/ci-templates/blob/1.33.0/.github/changelog-config.json -O default-config.json
```

Tag branches don't contain `blob` inside their path, you can test this with https://raw.githubusercontent.com/bakdata/ci-templates/1.32.0/.github/changelog-config.json
```suggestion
          wget -c https://raw.githubusercontent.com/bakdata/ci-templates/1.33.0/.github/changelog-config.json -O default-config.json
```